packages/type-orm/src/services/sorting.service.ts

```diff
@@ -31,7 +37,16 @@ export class SortingService {
 
     return sortingFields.reduce((acc: TypeORMOrdering, sortingField) => {
       const dbName = dbColumns[sortingField.name]
-      acc[`${prefix}${dbName}`] = sortingField.direction
+      acc[`${prefix}${dbName}`] = (sortingField.nulls) 
```

Comment:
It'd be prob simpler to just use ternary on the `nulls` property no? I guess that if you pass `undefined` or `null` to it, it does not do anywhing?

Would make it more readable. If `nulls` cannot receive undefined/null, nevermind this comment.

Reply:
done